x-pack/plugin/identity-provider/build.gradle

```diff
@@ -111,16 +112,16 @@ tasks.named("thirdPartyAudit").configure {
         'org.apache.velocity.runtime.resource.loader.StringResourceLoader',
         'org.apache.velocity.runtime.resource.util.StringResourceRepository',
         // [missing classes] OpenSAML depends on Apache XML security which depends on Xalan, but only for functionality that OpenSAML doesn't use
-        'org.apache.xml.dtm.DTM',
-        'org.apache.xml.utils.PrefixResolver',
-        'org.apache.xpath.Expression',
-        'org.apache.xpath.NodeSetDTM',
-        'org.apache.xpath.XPath',
-        'org.apache.xpath.XPathContext',
-        'org.apache.xpath.compiler.FunctionTable',
-        'org.apache.xpath.functions.Function',
-        'org.apache.xpath.objects.XNodeSet',
-        'org.apache.xpath.objects.XObject',
+        // 'org.apache.xml.dtm.DTM',
+        // 'org.apache.xml.utils.PrefixResolver',
+        // 'org.apache.xpath.Expression',
+        // 'org.apache.xpath.NodeSetDTM',
+        // 'org.apache.xpath.XPath',
+        // 'org.apache.xpath.XPathContext',
+        // 'org.apache.xpath.compiler.FunctionTable',
+        // 'org.apache.xpath.functions.Function',
+        // 'org.apache.xpath.objects.XNodeSet',
+        // 'org.apache.xpath.objects.XObject',
```

Comment:
Should we delete these now that we know they're not needed?

Reply:
good catch, pushed [4917d12](https://github.com/elastic/elasticsearch/pull/98199/commits/4917d1258200bc2719b3e16bacda414403f1f184)